### ISSUE TITLE
Product Elements: Adding “text styling toolbar” to Add to cart button

### ISSUE THREAD

**Issue by imanish003** [woocommerce/woocommerce-blocks#8860]
We wanna add following toolbar controls to Add to cart button toolbar, which is also available on core/button.
![image](https://user-images.githubusercontent.com/16707866/227856350-2fd07eb1-d847-4a7f-8168-9560b40aeea5.png)

For more info, check following GitHub discussion: https://github.com/woocommerce/woocommerce-blocks/discussions/8848

This was part of https://github.com/woocommerce/woocommerce-blocks/issues/8085, but I am creating this new issue to unblock https://github.com/woocommerce/woocommerce-blocks/issues/8085.
